components/dashboard/src/components/ConfirmationModal.tsx

```diff
@@ -26,7 +27,11 @@ export default function ConfirmationModal(props: {
     ];
 
     if (props.warningText) {
-        children.unshift(<Alert type="warning">{props.warningText}</Alert>);
+        children.unshift(
+            <Alert type="warning">
+                <strong>{props.warningHead}</strong>: {props.warningText}
+            </Alert>,
+        );
```

Comment:
This introduces a dependency between the `warningHead` and `warningText` props; a user of the component has to know to supply both props otherwise they'll end up with a ':' prepended to the `warningText`.

We should ensure that the component looks ok with and without a `warningHead` prop.

Reply:
Right, we can make condition on `:` for that, something like : `{(props.warningHead)?": ":""}`
WDYT @andrew-farries ?

Reply:
Yes, I think that would work.